scarlet2/morphology.py

```diff
@@ -37,12 +37,18 @@ class ProfileMorphology(Morphology):
     center: jnp.array
     size: float
     ellipticity: (None, jnp.array)
+    g: (None, jnp.array)
 
     def __init__(self, center, size, ellipticity=None, bbox=None):
 
         # define radial profile function
         self.center = center
         self.size = size
+        self.ellipticity = ellipticity
+
+        g_factor = 1 / (1. + jnp.sqrt(1. - (ellipticity[0]**2 + ellipticity[1]**2)))
```

Comment:
You need to check if `ellipticity is not None` for this to work. Please add.

Reply:
OK, I added the check if ellipticity is None in the `__init__` and set it to `jnp.zeros((2,))` if so